### PR TITLE
fix pango 1.40.2 install issue on OSX 10.6.x

### PR DESCRIPTION
**Pull request by ssreekanth**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

pango 1.40.2 fails to install on OSX 10.6.x with the following compilation errors -

```
pangocoretext-fontmap.c:122: error: initializer element is not constant
pangocoretext-fontmap.c:122: error: (near initialization for ‘ct_weight_map[0].ct_weight’)
pangocoretext-fontmap.c:131: error: initializer element is not constant
pangocoretext-fontmap.c:131: error: (near initialization for ‘ct_weight_map[9].ct_weight’)
make[4]: *** [pangocoretext-fontmap.lo] Error 1
```

This pull request fixes these compilation errors.

### CODE REVIEW

File: Formula/pango.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@ class Pango < Formula
     sha256 "1e9be870617caba7603eb1b8953eed684a863035ae2becabe0dd86b9f96a540a" => :mavericks
   end
 
+  patch :p1, :DATA
```
Comment (DomT4) [inline Formula/pango.rb]: Can you report this upstream & include a link to the upstream bug report above this line? Thanks!

Comment (ssreekanth) [inline Formula/pango.rb]: Looks like macports guys already reported [this issue](https://bugzilla.gnome.org/show_bug.cgi?id=770729) upstream.

**Comment by tschoonj** [comment]
I have opened a PR for pango 1.40.3 which includes the patch.

**Comment by zmwangx** [comment]
Merged #4772. Thanks anyway.

**Pull request closed without merging**